pyproject.toml

```diff
@@ -67,6 +67,13 @@ source = "vcs"
 [tool.hatch.build.hooks.vcs]
 version-file = "src/importnb/_version.py"
 
+[tool.hatch.build.targets.sdist]
+exclude = [
+  "docs/coverage",
+  "conftest.py"
```

Comment:
if the `conftest.py` is needed to actually run the tests, and no tests exist outside of docs, could it be moved to `docs`? Currently, downstream, I am bringing in:

```
      source_files:
        - conftest.py
        - docs
```

I guess ideally it would be only:

```
docs/
```

And not pull in any irrelevant stuff.

Reply:
i mved the conftest to the docs. so conftest is not at the root level anymore.

Reply:
that didn't work. it turned out the best solution was removing the conftest and initializing the plugin from the pytest config in pp.t